scripts/nodelib.py

```diff
@@ -10,14 +10,16 @@
 except NameError:
     pass
 
+rust_version = open(os.path.join(os.path.dirname(__file__), '../rust-toolchain'), 'r').read().strip()
+
 """Installs cargo/Rust."""
 def install_cargo():
     try:
         subprocess.call([os.path.expanduser('~/.cargo/bin/cargo'), '--version'])
     except OSError:
         print("Installing Rust...")
         subprocess.check_output('curl https://sh.rustup.rs -sSf | sh -s -- -y', shell=True)
-        subprocess.call([os.path.expanduser('~/.cargo/bin/rustup'), 'default', 'nightly'])
+        subprocess.call([os.path.expanduser('~/.cargo/bin/rustup'), 'default', rust_version])
```

Comment:
I don' think we need to change default at all. Since `cargo ...` overrides the default version anyway. Is it just to install it?

Reply:
Ah got it. This script assumes rustup exist, because that we don't need install at all. `cargo ...` will install it anyway if not exist